core/deployment/src/main/java/io/quarkus/deployment/ide/Ide.java

```diff
@@ -1,15 +1,19 @@
 package io.quarkus.deployment.ide;
 
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
+import io.quarkus.dev.console.DevConsoleManager;
+
 public enum Ide {
 
     // see for cli syntax of idea https://www.jetbrains.com/help/idea/opening-files-from-command-line.html
-    IDEA("idea", "--line %s", "--help"),
```

Comment:
The line parameter was used to directly point the cursor at a given line.
E.g. when clicking on a line of a stacktrace.

I remember testing this, and it worked for me.
Is there any reason to remove `--line` from the call?

Reply:
It didn't work for me, it just created an empty file called --line. 

Changing it to use the same format as the others worked for me.